plugins/builds/getLatestBuild.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const buildSchema = schema.models.build.get;
+const jobIdSchema = joi.string().regex(/^[0-9]+$/);
+const statusSchema = joi.reach(schema.models.build.base, 'status');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/builds/latest',
```

Comment:
would it be better to modify/combine with this endpoint? https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/jobs/latestBuild.js
I think that path makes more sense to me

Reply:
👍 , Completely missed this endpoint earlier

Reply:
Done